x/rollup/keeper/genesis.go

```diff
@@ -0,0 +1,25 @@
+package keeper
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/polymerdao/monomer/x/rollup/types"
+)
+
+// InitGenesis - Init store state from genesis data
+func (k *Keeper) InitGenesis(ctx sdk.Context, state types.GenesisState) { //nolint:gocritic // hugeParam
```

Comment:
can we return errors here? Same for `ExportGenesis`

Reply:
We could add logging here to add additional messenging for the user, however we need to panic instead of throwing an error to properly implement the Cosmos SDK module `HasGenesis`. Unfortunately this is how other Cosmos SDK modules handle their `InitGenesis` and `ExportGenesis` cases.

Reply:
Got it, no worries then.